pkg/offline/offline.go

```diff
@@ -347,7 +347,7 @@ func CountHeartbeats(filepath string) (int, error) {
 
 		_ = tx.Rollback()
 
-		return count, err
+		return 0, err
```

Comment:
If there's an error then `Count()` would have already returned `0`.

Reply:
In Go when error happens we would not trust/reuse the returned response. Think if the lib changes its behaviour to not return `0` and return `-1` for example.

Reply:
I also improved code by defering rollback.
https://github.com/wakatime/wakatime-cli/compare/59ac5c08cb13a47e839988dded1fc15d21bb3083..00987f5fa86d8e8d911185c2210097d1aa03211d